docs/modules/ROOT/pages/operations/browser-sso.adoc

```diff
@@ -1,21 +1,21 @@
 :description: Single Sign-On (SSO) for Neo4j Browser.
 
-
+[role=enterprise-edition]
 [[browser-sso]]
 = Browser Single Sign-On
 
-Neo4j Browser provides basic support for some Single Sign-On (SSO) providers under specific Enterprise environment configurations.
-Contact Neo4j Professional Services for additional assistance (see link:https://neo4j.com/professional-services/[_https://neo4j.com/professional-services/_^] for more information about Professional Services).
-Your organization needs an existing compatible SSO provider (local or external) already configured.
+Neo4j Browser provides support for Single Sign-On (SSO) providers under specific Enterprise environment configurations when used with (self-hosted) Neo4j Server v4.4 and later.
+Additionally, you need a compatible SSO provider (local or external) already configured.
+For more information on the configuration, see link:https://neo4j.com/docs/operations-manual/current/authentication-authorization/sso-integration/#auth-sso-parameters[Operations Manual -> OIDC configuration settings^].
```

Comment:
This link seems broken to me, is that just because it hasn't been published yet?

Reply:
Correct, it'll work once the 4.4 docs are published.